docsrc/docextract-intro.rst

```diff
@@ -0,0 +1,80 @@
+    result = client.call(RetrieveStorageObject(extracted_file.result))
+
+In this example, given the path to a document, we called DocumentExtraction with a single file and waited for the result.
+With most use cases, this is all you will need to do. The returned 'result' will either be a dictionary (if
+document-level text and/or metadata are requested) or a list of dictionaries (where each dictionary contains
+the text for a unique page/block).
+
+``DocumentExtraction`` is highly configurable. You can customize settings at the document, page, block, token or
+character level. You can also choose from a selection of preset configurations. You configure DocumentExtraction
+by passing in a Python dictionary or JSON string.
+
+Here's an Example::
+
+    my_ocr_config = {
+        "preset_config": "standard"
+    }
+
+    job = client.call(DocumentExtraction(files=['./path_to_doc.pdf'], json_config=my_ocr_config))
+
+
+Preset Configurations
+=====================
+
+Setting::
+
+    "preset_config": "simple"
+    "preset_config": "legacy"
+    "preset_config": "detailed"
+    "preset_config": "ondocument"
+    "preset_config": "standard"
+
+Five preset ocr configurations are provided: ``legacy``, ``simple``, ``ondocument``, ``standard``,
+and ``detailed``.
+
+Most users will only need to use "standard" to get both document and page-level text and block positions in
+a nested response format (returned object is a nested dictionary).
+
+The "simple" configuration provides a basic and fast (3-5x faster) OCR option for native PDFs- i.e. it will
+*not* work with scanned documents. Returns document, page, and block-level text and the returned object is a
+nested dictionary.
+
+The "legacy" configuration is principally intended for users who ran Indico's original pdf_extraction function to extract
+text and train models. Use "legacy" if you are adding samples to models that were trained with data using
+the original pdf_extraction. It returns a dictionary containing only the extracted text at the document-level.
+
+The "detailed" configuration provides OCR metrics and details down to the character level- it's a lot of data.
+In addition to document, page, and block-level text, it provides information on the text font/size,
+confidence level for extracted characters, alternative characters (i.e. second most probable character), character
+position information, and more. It returns a nested dictionary.
+
+"ondocument" provides similar information to "detailed" but does not include text/metadata at the
+document-level. It returns a list of dictionaries where each dictionary is page data. This preset configuration
+also supports an additional "vdp" attribute, which, if set to True, performs an additional block classification
+and association step. The former adds a "label_type" to every block, and the latter adds an "associations"
+attribute to the result (located on the first page).
```

Comment:
The end user probably doesn't need to know which additional info is added by vdp in which of two additional steps. Probably they just need to know that vdp adds label_type to each block and adds associations. (Is associations added as a document-level attribute? Or is it an additional attribute just in the first page or block of the document?).

Also what does "vdp" stand for? If it's something meaningful then that will help with comprehension, instead of just trying to remember three letters.

Reply:
Building on Liz's feedback, vdp details are also not present in any of the doc extraction configuration documentation, including the ondocument default config.